pages/api/performances/nominate.js

```diff
@@ -36,28 +36,37 @@ export default async (req, res) => {
   // TODO for now we assume frontend will filter correctly and only
   // eligible awards will show up for validation
   try {
+    // First clear all the current nominations
+    await prisma.awardPerformance.deleteMany({
+      where: {
+        performance_id: parseInt(performanceID),
+        user_id: parseInt(judgeID) || currentUserID,
+      },
+    });
```

Comment:
@feng-eric @thepenguinco We should clear the current nominations before updating to the new ones. I thought about whether this would raise problems for finalized awards, but I think this should be desired behaviour, as overwriting a finalized nomination should be allowed.

We also need to unfinalize the award in `Awards` if a finalized nomination is deleted.

Reply:
Note: 

Disallow deleting nominations if award is finalized. Handle in API, and display error message on frontend.

cc @thepenguinco

Reply:
@OustanDing For unfinalizing an award if the nomination is deleted, it should be handled here right?

Reply:
@OustanDing I'm a bit confused. Are we going to allow finalized nominations or no

Reply:
If so, shouldn't this be:

`await prisma.awardPerformance.deleteMany({
      where: {
        performance_id: parseInt(performanceID),
        user_id: parseInt(judgeID) || currentUserID,
        status: 'NOMINEE'
      },
    });
`

Reply:
If you're asking about whether we should be able to _delete_ finalized nominations, we are not allowing that. However, we shouldn't just delete the nominations with `status='NOMINEE'`, we should raise an error that blocks them from making any changes at all on the backend. On the frontend, any inputs (dropdowns, text input) for finalized nominations should be **disabled**. I will make those changes when your and Mayank's changes go in.

Reply:
Ah, got it! Makes sense